Scripts/GetAutoShutdownPolicyStatus/README.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+When you manage multiple labs, it is sometimes important to make sure that all your labs have the Auto-Shutdown policy enabled.  This can be a tedious exercise if you have many labs to examine.  To automate this process, we have introduced a PowerShell script that will look at all your labs and verify they are opted-in to the Auto-Shutdown lab policy. 
+
+The syntax of the script is as follows:
+```
```

Comment:
Add `powershell` after ` ``` `, as in ` ```powershell`, to provide syntax highlighting for the examples.

For example,
```
` ` `powershell
GetAutoShutdownPolicyStatus.ps1 [-SubscriptionIds] [-LabName] [-Verbose]
` ` `
```
turns into
```powershell
GetAutoShutdownPolicyStatus.ps1 [-SubscriptionIds] [-LabName] [-Verbose]
```